### PR TITLE
feat: support vpc and cvm

### PR DESCRIPTION
**Pull request by tongyiming**
### Description of your changes

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: examples/vpc/resource.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: vpc.tencentcloud.jet.crossplane.io/v1alpha1
+kind: VPC
+metadata:
+  name: example-vpc
+spec:
+  forProvider:
+    cidrBlock: "10.1.0.0/16"
+    name: "test-crossplane"
```
Comment (haarchri) [inline examples/vpc/resource.yaml:8]: Can we skip name and can use metadata.name?

File: examples/null/resource.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: null.template.jet.crossplane.io/v1alpha1
+apiVersion: null.tencentcloud.jet.crossplane.io/v1alpha1
```
Comment (haarchri) [inline examples/null/resource.yaml:1]: Did we need null resource in example folder ? Think we can remove it

File: examples/cvm/resource.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+kind: Instance
+metadata:
+  name: example-cvm
+spec:
+  forProvider:
+    instanceName: "test-cvm"
+    availabilityZone: "ap-guangzhou-3"
+    imageId: "img-9qrfy1xt"
+    instanceType: "S3.MEDIUM2"
+    vpcId: "vpc-eorr0tjj"
```
Comment (haarchri) [inline examples/cvm/resource.yaml:11]: Can we use here vpcIdRef If implemented ?

**Review by haarchri — COMMENTED** [review]
Thanks for Implementation - added a few small comments

File: examples/vpc/resource.yaml (hunk 1 above)
Comment (tongyiming) [reply examples/vpc/resource.yaml:8]: yes, but name is required. how to set name with metadata.name and skip name?

Comment (haarchri) [reply examples/vpc/resource.yaml:8]: https://github.com/crossplane/terrajet/blob/main/docs/configuring-a-resource.md#case-1-name-as-external-name-and-terraform-id can you check here ?

File: examples/null/resource.yaml (hunk 2 above)
Comment (haarchri) [reply examples/null/resource.yaml]: Did we need null resource ? Think you can remove

Comment (tongyiming) [reply examples/null/resource.yaml]: I check null had been deleted.
image

**Pull request merged**